src/mvPlotting.cpp

```diff
@@ -2703,21 +2996,32 @@ DearPyGui::set_configuration(PyObject* inDict, mvPlotAxisConfig& outConfig, mvAp
 	if (inDict == nullptr)
 		return;
 
+	if (PyObject* item = PyDict_GetItemString(inDict, "scale")) outConfig.scale = ToInt(item);
+	if (PyObject* item = PyDict_GetItemString(inDict, "formatter")) outConfig.formatter = ToString(item);
+
 	// helper for bit flipping
 	auto flagop = [inDict](const char* keyword, int flag, int& flags)
 	{
 		if (PyObject* item = PyDict_GetItemString(inDict, keyword)) ToBool(item) ? flags |= flag : flags &= ~flag;
 	};
 
 	// axis flags
+	flagop("no_label", ImPlotAxisFlags_NoLabel, outConfig.flags);
 	flagop("no_gridlines", ImPlotAxisFlags_NoGridLines, outConfig.flags);
 	flagop("no_tick_marks", ImPlotAxisFlags_NoTickMarks, outConfig.flags);
 	flagop("no_tick_labels", ImPlotAxisFlags_NoTickLabels, outConfig.flags);
-	flagop("log_scale", ImPlotAxisFlags_LogScale, outConfig.flags);
```

Comment:
`log_scale`: I think we can retain this for backward compatibility, but declare as deprecated (see my comments on `DEPRECATED_KEYWORD_ARG` in the PR discussion - it should spit a warning, but otherwise work as usual). I believe `log_scale` is used a lot, and simply removing it will be too breaking. We can set `outConfig.scale` to `ImPlotScale_Log10` on input, and check if it's set to that value on output (in `fill_configuration_dict`).

The same applies to `time`: we can retain it, setting or checking for `ImPlotScale_Time` in `config.scale`. I bet this one is even more popular than `log_scale`.